test/controllers/auth/AuthActionSpec.scala

```diff
@@ -21,24 +21,29 @@ import controllers.routes
 import helpers.ErsTestHelper
 import org.mockito.ArgumentMatchers
 import org.mockito.Mockito._
+import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{Assertion, BeforeAndAfterEach}
 import play.api.Play
 import play.api.http.Status
 import play.api.mvc.{BodyParsers, Result, Results}
 import play.api.test.FakeRequest
+import play.api.test.Helpers.{contentAsString, defaultAwaitTimeout, status}
 import uk.gov.hmrc.auth.core.AuthProvider.GovernmentGateway
 import uk.gov.hmrc.auth.core._
 import uk.gov.hmrc.auth.core.authorise.Predicate
 import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals.allEnrolments
 import uk.gov.hmrc.domain.EmpRef
-import uk.gov.hmrc.play.test.{UnitSpec, WithFakeApplication}
+import org.scalatest.{Matchers, OptionValues, WordSpecLike}
+import org.scalatestplus.play.guice.GuiceOneServerPerTest
 
 import scala.concurrent.Future
 
-class AuthActionSpec extends UnitSpec with ErsTestHelper with BeforeAndAfterEach with WithFakeApplication {
+class AuthActionSpec extends WordSpecLike with Matchers with OptionValues
```

Comment:
What's this OptionValues dependency for? I'm not familiar with it.

Reply:
I'm not crazy certain either, it's just a direct replacement of UnitSpec alongside WordSpecLike and Matchers